lib/src/DatePicker/DatePickerToolbar.tsx

```diff
@@ -18,7 +18,7 @@ export const useStyles = makeStyles(
       top: 4,
     },
   },
-  { name: 'MuiPickersDatePickerRoot' }
+  { name: 'MuiPickersDatePickerToolbar' }
```

Comment:
Maybe we need some reusable function that will build a standardized name. Something like
```ts
const buildStyleName = (component: React.ComponentType<any>) => {
   return `MuiPickers${component.displayName || component.name}`
}
```

@oliviertassinari what do your think?

Reply:
How about making the second argument to `makeStyles()` a component?

Reply:
I think that we should drop the "Pickers" prefix to match the exported name. What would be wrong with?

```suggestion
  { name: 'MuiDatePickerToolbar' }
```

Actually, this would be even better as the component is private:

```suggestion
  { name: 'PrivateDatePickerToolbar' }
```

Reply:
What do you mean by "even better"?
#1707 makes this public

Reply:
Also regarding the `buildStyleName()` function, especially if the prefix is now just `Mui` and nost `MuiPickers`, I think this library should just do the same as the main material-ui library. Does that one have such function?

Reply:
> #1707 makes this public

I think that the change should be atomic. If we make a new component public, it should be done all at once. This would avoid the cases were we forget/revert/etc.

Reply:
> Does that one have such function?

Great point. We don't, I think that a string duplication is good enough